frontend/src/mixins/Lists.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Vue } from "vue-property-decorator";
+
+@Component
+export default class Lists extends Vue {
+  lists: string[] = [];
+  listsBasePath: string = "files";
```

Comment:
For sound editing we're working on tasks, not files.

Reply:
The vue component that uses the mixin can override one or more of the properties of the mixin  https://vuejs.org/v2/guide/mixins.html#Option-Merging. Also I realize now it should be named something generic to the firebase shallow query since there's no reason it should be restricted to lists

Reply:
So are you going to make it more generic? I could not get this.

Reply:
Oh I see that you've already done this.